scss/_variables.scss

```diff
@@ -43,9 +43,9 @@ $grays: (
 $accessible-orange: #f16e00 !default;
 $brand-orange:      #ff7900 !default;
 //// Functional colors
-$functional-green:  #32c832 !default;
-$functional-blue:   #527edb !default;
-$functional-yellow: #fc0 !default;
+$functional-green:  #228722 !default;
+$functional-blue:   #4170d8 !default;
+$functional-yellow: #8f7200 !default;
```

Comment:
Could be temporary. Kept it just to see where it's implemented and where it's not.